source/Junkership.js

```diff
@@ -3,43 +3,62 @@ var Keyb = require("keyb")
 
 import Reference from "./Reference.js"
 import Projectile from "./Projectile.js"
+import Score from "./Score.js"
 
 var junkerTex = Pixi.Texture.fromImage(require("./images/blue-starship.png"))
 
 export default class Junkership extends Pixi.Sprite {
     constructor() {
         super(junkerTex)
-        this.speed = 60
-        this.score = 0
         game.playerCount++
+        this.speed = 60
+        this.score = new Score()
     }
     update(delta) {
-        if(Keyb.isDown("<up>") && this.position.y > 0) {
-            this.position.y -= this.speed * delta
-        }
-        if(Keyb.isDown("<down>") &&
-            (this.position.y + this.height) < Reference.GAME_HEIGHT) {
-            this.position.y += this.speed * delta
+        var relativeSpeed = this.speed * delta
+        if(Keyb.isDown("<up>")) {
+            this.move(-relativeSpeed, "y")
+        } else if(Keyb.isDown("<down>")) {
+            this.move(relativeSpeed, "y")
```

Comment:
So this is always an interested gamedev question: what is the intended behavior here? If I hold UP, I should move UP, if I hold DOWN, I should move DOWN. But what if I hold both at the same time?

Currently, it's programmed that you... go UP? I think you should just not move at all. If we just remove that `else`, it means we're trying to move UP and DOWN at the same time, which means we don't move anywhere.

But what are your thoughts? What should happen here?